internal/zinc-scripted/src/main/contraband-scala/sbt/internal/inctest/Project.scala

```diff
@@ -7,22 +7,23 @@ package sbt.internal.inctest
 final class Project private (
   val name: String,
   val dependsOn: Vector[String],
-  val in: Option[java.io.File]) extends Serializable {
+  val in: Option[java.io.File],
+  val scalaVersion: Option[String]) extends Serializable {
   
-  private def this(name: String) = this(name, Vector(), None)
+  private def this(name: String) = this(name, Vector(), None, None)
```

Comment:
We intentionally include it because there's generated Scala code could still be read by ppl via GitHub etc without understanding the original GraphQL syntax.